### DISCUSSION TITLE
ALEGoToDefinition, how to go back to the function call?

### DISCUSSION THREAD

**Discussion by ubaldot** [w0rp/ale#4073] [Q&A - Ask for help with problems] [answered]
Hello!
Say that I am working on `file1.c` that contains a call to a function `foo` for which I want to see its definition.
All I have to do is to place the cursor over the function call `foo` and run `:ALEGoToDefinition`. Perfect. 
But then, after I looked the `foo` definition, I want to go back to my initial point, namely I want to go back to `file1.c` with the cursor placed on the function call `foo`. 
How to do that?

**Comment by hsanson** [comment, marked as answer]
Normally I use "ctrl-o".  See `:h ctrl-o` or better `:h jump-motions` for more details.
